modules/library/api/src/main/java/org/geotools/styling/Stroke.java

```diff
@@ -246,8 +256,20 @@ public Graphic getGraphicStroke() {
     void setLineCap(Expression lineCap);
 
     /**
-     * This parameter encodes the dash pattern as a seqeuence of floats.<br>
-     * The first number gives the length in pixels of the dash to draw, the
+     * Shortcut to retrieve dash array in the case where all expressions
+     * are literal numbers. Return the default value if one of the expressions
+     * is not a literal.
+     */
+    float[] getDashArray();
```

Comment:
I would just deprecate this one and ask people to switch to the Expression based version